CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contributing to Altruism in tech
+
+Thank you for your interest in contributing to **Altruism in Tech** :tada:! 
+If you're interested in contributing, you should find all of the details here. We've centralized it into this repository, we'd like to keep things tidy and focused. Issues and roadmap goals will reflect changes across multiple repositories on Github, so you'll have the opportunity to contribute to multiple aspects of this project.
+
+**Altruism in tech** is promoting diversity and inclusion in tech by connecting newcomers with tech professionals.
+
+This document is a set of guidelines for contributing to **Altruism in tech** on GitHub. These are guidelines, not rules. This guide is meant to make it easy for you to get involved.
+
+* [Participation guidelines](#participation-guidelines)
+* [What we're working on](#what-were-working-on)
+* [How to submit changes](#how-to-submit-changes)
+* [How to report bugs](#how-to-report-bugs)
+* [Communication channels](#communication-channels)
+
+## Participation guidelines
+
+This project adheres to a [Mozilla Community Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/). By participating, you are expected to uphold this code. Please report unacceptable behavior to ipsha21@gmail.com.
+
+## What we're working on
+
+Take a look at the issues in our [current milestone](https://github.com/ipsha21/altruism-in-tech/issues) or browse for some good first bugs to get started! 
+
+## How to submit changes
+
+Once you've identified one of the issues above that you feel you can contribute to, you're ready to make a change to the project repository!
+ 
+ 1. **[Fork](https://help.github.com/articles/fork-a-repo/) this repository**. This makes your own version of this project you can edit and use.
+ 2. **[Make your changes](https://guides.github.com/activities/forking/#making-changes)**! You can do this in the GitHub interface on your own local machine. Once you're happy with your changes...
+ 3. **Submit a [pull request](https://help.github.com/articles/proposing-changes-to-a-project-with-pull-requests/)**. This opens a discussion around your project and lets the project lead know you are proposing changes.
+
+ First time contributing to open source? Check out this *free* series, [How to Contribute to an Open Source Project on GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github).
+
+## How to report bugs
+
+Notice a mistake? Please file any bugs, requests, or questions in our [issue tracker](https://github.com/ipsha21/altruism-in-tech/issues)!
+
+## Communication channels
+
+If you have anything to share, anything at all, feel free to drop by the Altruism in Tech [room](https://gitter.im/AltruismInTech/Lobby) on Gitter and say hi. We would love to hear from you. You can also contact Ipsha by [email](ipsha21@gmail.com) or on [Twitter](https://twitter.com/ipsha21).
```

Comment:
> You can also contact Ipsha by` [email](ipsha21@gmail.com)`

Clicking this takes to a 404 github page. Please modify it to open a mailto: dialog box. Thanks!